.teamcity/_self/lib/customBuildType/E2EBuildType.kt

```diff
@@ -147,10 +147,10 @@ open class E2EBuildType(
 					set -x
 
 					mkdir -p screenshots
-					find test/e2e -type f -path '*/screenshots/*' -print0 | xargs -r -0 mv -t screenshots
+					find test/e2e/results -type f \( -iname \*.webm -o -iname \*.png \) -print0 | xargs -r -0 mv -t screenshots
```

Comment:
This is taken from the helpful [Stackoverflow](https://unix.stackexchange.com/a/15309) post.